### PR TITLE
Switch license to CDLA-Permissive 2.0

### LINKED ISSUE #4: HPSF: Data License Change
As part of joining the [HPSF](https://hpsf.io/), we might want to unify the licensing here.

In our draft technical charter, we write that we would license data under CDLA-Permissive 2.0 License, available at https://cdla.dev/permissive-2-0.

### PR DESCRIPTION
**Pull request by roelof-groenewald**
Closes #4. 

All contributors please confirm that you are okay with this license change by checking the box below:

- [x] @roelof-groenewald 
- [ ] @nruof 
- [x] @ax3l

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +0,0 @@
...
-please contact Berkeley Lab's Innovation & Partnerships Office at
-IPO@lbl.gov.
-
-NOTICE.  This Software was developed under funding from the U.S. Department
-of Energy and the U.S. Government consequently retains certain rights. As
-such, the U.S. Government has been granted for itself and others acting on
-its behalf a paid-up, nonexclusive, irrevocable, worldwide license in the
-Software to reproduce, distribute copies to the public, prepare derivative
-works, and perform publicly and display publicly, and to permit other to do
-so.
```
Comment (roelof-groenewald) [inline README.md:38]: @ax3l how much of this should we keep with the license switch?

Comment (ax3l) [reply README.md:38]: Nothing. The old text was the software license, which does not apply for this repo.

File: README.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Overview
+
+This repository contains data files that are too large for the main [WarpX source directory](https://github.com/ECP-WarpX/WarpX).
+
+## Documentation
+
+This section describes the directory structure:
+
+...
```
Comment (ax3l) [inline README.txt]: @roelof-groenewald can you complete this section and make this again a `.md` file? :)

**Comment by ax3l** [comment]
@roelof-groenewald please ping every contributor to this data repo and ask them to comment here that they are ok with the license change for their contributions :pray:

**Comment by ax3l** [comment]
I confirm I am ok with the license change. Thank you! :+1:

**Pull request merged**